helm/prometheus-rules/templates/alerting-rules/kyverno.rules.yml

```diff
@@ -0,0 +1,26 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+    cluster_type: "management_cluster"
+  name: kyverno.management_cluster.rules
```

Comment:
Why not `kyverno.rules`? Or do we have this naming anywhere else? I thought it was just the filename

Reply:
honestly I don't know what that `name` field does at all. I followed the structure I found here https://github.com/giantswarm/prometheus-rules/blob/master/helm/prometheus-rules/templates/alerting-rules/kvm.management-cluster.rules.yml#L9

Reply:
The only thing I see is, that it matches the file name - so `kyverno.rules` seems correct to me.

Reply:
changed